ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/Client.java

```diff
@@ -280,6 +280,24 @@
    */
   CompletableFuture<ConnectorDescription> describeConnector(String connectorName);
 
+  /**
+   * Define a session variable which can be referenced in sql commands by wrapping it with ${}
+   *
+   * @param variable the name of the variable
+   * @param value the value represented by the variable
+   */
+  void define(String variable, Object value);
```

Comment:
Can we add an integration test for a non-string value? It's not clear to me from the docs whether variable substitution is even supported for non-string values. This interface definition suggests that is, and we should have corresponding test coverage.